CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.15.0-rc.0 / 2019-12-18
+
+* [CHANGE] Discovery: Removed `prometheus_sd_kubernetes_cache_*` metrics. #6393
+* [FEATURE] API: Added new endpoint for exposing per metric metadata `/metadata`. #6420 #6442
+* [FEATURE] Remote write: Added config option to name remote target which is added as another label dimension to `prometheus_remote_storage_*` metrics. #6043
+* [FEATURE] Metrics format: Added examplar support to OpenMetrics parser. #6292
+* [FEATURE] Remote write: Added `prometheus_remote_storage_sent_bytes_total` counter per queue. #6344
+* [ENHANCEMENT] TSDB: Reduced memory footprint of loaded TSDB blocks. #6418 #6461
+* [ENHANCEMENT] TSDB: Significantly optimized what we buffer during compaction which should result in lower memory footprint during compaction. #6422 #6452 #6468 #6475
+* [ENHANCEMENT] promql: Improved query evaluation performance. #6427
```

Comment:
This only reduces GC a bit, this wording makes it seem like it's much bigger. Thus may not even be worth mentioning.

Reply:
yup, was thinking about this.